src/Component/QuantityInput/QuantityInput.js

```diff
@@ -1,26 +1,37 @@
-import React from 'react'
+import PropTypes from 'prop-types';
 
 const QuantityInput = ({
-    quantity,
-    onDecrementClick,
-    onIncrementClick,
-    minValue = Number.MIN_SAFE_INTEGER,
-    maxValue = Number.MAX_SAFE_INTEGER,
+  quantity,
+  onDecrementClick,
+  onIncrementClick,
+  minValue = Number.MIN_SAFE_INTEGER,
+  maxValue = Number.MAX_SAFE_INTEGER,
 }) => (
-    <span className="product-quantity">
-        <button 
-            onClick={()=> 
-                onDecrementClick()}
-            disabled={quantity <= minValue}
-        >-
-        </button>
-        <input type="text" value={quantity} readOnly />
-        <button 
-            onClick={()=> onIncrementClick()}
-            disabled={quantity > maxValue}
-        >+
-        </button>
-    </span>
-)
+  <span className='product-quantity'>
+    <button onClick={() => onDecrementClick()} disabled={quantity <= minValue}>
+      -
+    </button>
+    <input type='text' value={quantity} readOnly />
+    <button onClick={() => onIncrementClick()} disabled={quantity > maxValue}>
+      +
+    </button>
+  </span>
+);
 
-export default QuantityInput
\ No newline at end of file
+QuantityInput.propTypes = {
+  quantity: PropTypes.number,
+  onDecrementClick: PropTypes.func,
```

Comment:
Not typesafe. Make it required or add an extra check.
If handler is absent, make `null` as fallback value